examples/controls/README.md

```diff
@@ -0,0 +1,362 @@
+This project is bootstrapped with [Create Elm App](https://github.com/halfzebra/create-elm-app).
```

Comment:
BTW, I have tried this example with a Gear VR device, and I have achieved to see the controller in the scene. To accomplish this task I had to add `components/LaserControls.elm` and `components/Controls.elm` but in both of them, I have defined a type called `Hand`, I should refactor those types and put them in a separate file or folder, but I don't know yet where.